src/kit.ts

```diff
@@ -595,7 +612,11 @@ export class KitManager implements vscode.Disposable {
    * selection, the current kit is kept. The only way it can reset to `null` is
    * if the active kit becomes somehow unavailable.
    */
-  async selectKit(): Promise<Kit|null> {
+  async selectKit(): Promise<Kit | null> {
+    if (this._kits.length == 0) {
+      return null;
```

Comment:
I think this should be returned as a `Promise`:
```ts
return return Promise.resolve(null);
```

Reply:
Style guide question

Reply:
`async` auto-wraps returns in promises, so it should just be `null`.